src/libcollections/btree/set.rs

```diff
@@ -74,53 +74,91 @@ pub struct BTreeSet<T> {
     map: BTreeMap<T, ()>,
 }
 
-/// An iterator over a BTreeSet's items.
+/// An iterator over a `BTreeSet`'s items.
```

Comment:
a `BTreeSet`'s item**S**?

EDIT: Actually, it can be understood differently. Well, leaving this comment here anyway.